tests/drivers/uart/uart_async_api/testcase.yaml

```diff
@@ -11,14 +11,28 @@ tests:
     harness: ztest
     harness_config:
       fixture: gpio_loopback
-    depends_on: gpio
+    depends_on: gpio uart
```

Comment:
I don't think the uart keyword is required

Reply:
Removed